packages/ddex/client/package.json

```diff
@@ -47,23 +41,15 @@
     "react-router-dom": "6.20.0",
     "react-spring": "8.0.27",
     "react-use": "17.4.2",
-    "react-use-measure": "2.1.1"
+    "react-use-measure": "2.1.1",
+    "web3": "4.3.0"
```

Comment:
@theoilie noticed web3 (required for sdk) wasn't in the package.json but it used to be - did you intentionally remove it?

Reply:
not that i remember. it was probably an innocent bystander in my earlier fight against the monorepo to get a different trpc version